main.rb

```diff
@@ -0,0 +1,55 @@
+require_relative 'app'
+require_relative 'input_module'
+
+class Main
+  attr_reader :app
+
+  def initialize
+    @app = App.new
+  end
+
+  include InputModule
+
+  def menu_options
+    "
+M E N U
+1 - List all books
+2 - List all people
+3 - Create a person
+4 - Create a book
+5 - Create a rental
+6 - List all rentals for a given person id
+7 - Exit
+Please choose an option by entering a number: "
+  end
+
+  def menu
+    loop do
+      input = numeric_input(menu_options, (1..7))
+      case input
+      when 1
+        app.list_books
+      when 2
+        app.print_person
+      when 3
+        app.add_person
+      when 4
+        app.create_book
+      when 5
+        app.create_rental
+      when 6
+        app.list_rentals
+        break
```

Comment:
I'm trying to exit from the project but am unable to do so as it's missing the exit point. Please add an exit case as well. 👍🏻

Reply:
@zainsadaqat 
 😵‍💫 no problem I will do that. Thank you so much for your review. Can you star my projects and do interact with me on the following platforms.

👤 **ABDUL ALI**

> * This is My **GitHub**: ![GitHub followers](https://img.shields.io/github/followers/ABDULALI3468?label=ABDULALI&style=social)
> * This is My **Linkedin**: [![Linkedin Badge](https://img.shields.io/badge/-ABDUL%20ALI-blue?style=flat-square&logo=Linkedin&logoColor=white&link=https://www.linkedin.com/in/abdul-ali-5400bb216/)](https://www.linkedin.com/in/abdul-ali-5400bb216/)&nbsp;
> * This is My **Twitter**: [![Twitter Badge](https://img.shields.io/badge/-@mrabdul_ali_-1ca0f1?style=flat-square&labelColor=1ca0f1&logo=twitter&logoColor=white&link=https://twitter.com/mrabdul_ali)](https://twitter.com/mrabdul_ali)&nbsp;